examples/updated-vm-with-contracts/actions/call_test.go

```diff
@@ -0,0 +1,127 @@
+package actions
+
+import (
+	"context"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/hypersdk/chain/chaintest"
+	"github.com/ava-labs/hypersdk/codec/codectest"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/storage"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/x/contracts/runtime"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCallAction(t *testing.T) {
+	addr := codectest.NewRandomAddress()
+	counterBytes, err := LoadBytes("counter.wasm")
+	require.NoError(t, err)
+
+	// this will be the same when the contract is deployed
+	counterID := sha256.Sum256(counterBytes)
+	expectedAddress := storage.GetAccountAddress(counterID[:], []byte{0})
```

Comment:
we should recalculate the `expectedAddress` since this will change if we update the `counter` code. Otherwise this test will fail every time we change the api

Reply:
Sorry, my comment wasn't clear. I was referring to `[]byte{0}`. I'm being a little pedantic, but tests get a lot easier to read if you follow this pattern. That also means you can name the constant based on purpose, which helps readability